modules/concepts/overrides.md

```diff
@@ -144,6 +144,36 @@ class FrontController extends FrontControllerCore {
 }
 ```
 
+## Override a module
+
+Since version 1.6.0.11 of PrestaShop, a new feature is available that allows developers to override some parts of the modules.
+Despite a module having different files, the main one, additional classes, etc., you can override the main file, and ModuleFrontController (note: this is available since 1.7.2.0) based files.
+
+### Override module's main class by extending it
+
+To override a module's instance class, you have to extend it, giving the extended class the same name and adding the `Override` suffix:
+
+```php
+
+if (!defined('_PS_VERSION_')) {
+	exit;
+}
```

Comment:
```suggestion
```

Reply:
You don't need to specify this part, as we don't specify `<?php`.
It's only needed if you don't want to execute the whole file, but since it's a class declaration it's not mandatory.